app/script/view_model/AuthViewModel.coffee

```diff
@@ -205,6 +205,23 @@ class z.ViewModel.AuthViewModel
     $(".#{element_id}").show()
 
   _init_base: ->
+    @_init_url_parameter()
+    @_init_url_hash()
+
+    $(window)
+      .on 'dragover drop', -> false
+      .on 'hashchange', @_on_hash_change
+      .on 'keydown', @keydown_auth
+
+    $("[id^='wire-login'], [id^='wire-register'], [id^='wire-verify']").prevent_prefill()
+
+    # Select country based on location of user IP
+    @country_code (z.util.CountryCodes.get_country_code($('[name=geoip]').attr 'country') or 1).toString()
+    @changed_country_code()
```

Comment:
and `view_model` is not being used anywhere in that function.. so we might have to do something about it...